tests/Odbo/OdboTestCase.php

```diff
@@ -13,19 +13,19 @@ class OdboTestCase extends TestCase
 {
     use ResetsDatabase;
 
-	/**
-	 * @var \ODBO
-	 */
-	protected $testModel;
-
-	protected function setUp(): void
-	{
-		parent::setUp();
+    /**
+     * @var \ODBO
+     */
+    protected $testModel;
```

Comment:
Can we provide a type for this property rather than declare it in the docblock?
```suggestion
    protected \ODBO $testModel;
```

Reply:
[df5dc55](https://github.com/MaloufSleep/obray/pull/28/commits/df5dc55a7dd4809673aec997300f8d081f768721)